schema/schema.go

```diff
@@ -2,20 +2,23 @@ package schema
 
 import (
 	"fmt"
+	"strings"
 )
 
 type TypeStruct struct {
-	Fields         map[string]*StructField
-	Representation StructRepresentation
+	Fields         map[string]*StructField `json:"fields"`
+	Kind           string                  `json:"kind"`
```

Comment:
Ah, I see why you add that type there...

We could also implement the JSONMarshaler interface by adding a `MarshalJSON()` method to this type: https://golang.org/pkg/encoding/json/#Marshaler

example: https://golang.org/pkg/encoding/json/#example__customMarshalJSON